opcodes.c

```diff
@@ -42,18 +42,18 @@
 #define RX_ADDRESS ((OPCODE & 7) + 8 * ((PSW & (PSWMASK_RS0|PSWMASK_RS1))>>PSW_RS0))
 #define CARRY ((PSW & PSWMASK_C) >> PSW_C)
 
-static int read_mem(struct em8051 *aCPU, int aAddress)
+static int read_mem(struct em8051 *aCPU, int address)
```

Comment:
Why rename the parameter?

Reply:
The "a" prefix is for argument scope, makes it easier to read the code seeing where the variables come from.

Reply:
Oh I see. That explains the `aCPU` that I didn't understand. I was searching at first for a `bCPU` but didn't find any, so I thought that it meant "a CPU"😄 

Note, I usually don't like to prefix variables. And if I really need to, I'm usually changing the local ones, as they tend to have a much shorter scope. 

Yet, your repo, your rules 😉. Changed it in c7418ac3612b94e63e89574bb0c702090a3b4708.

Reply:
I typically follow this convention in my personal code: aArgument gGlobal mMember variable Class CONSTANT. Makes it easy to figure things out from short snippets of code. I've seen some people get confused about the prefixes and wanting to do things like anObject, though =)